lib/tasks/backfill_bank_data.rake

```diff
@@ -0,0 +1,13 @@
+namespace :bank_account do
+  desc "Backfill account number to new encrypted column"
+  task backfill: :environment do
+    BankAccount.find_each do |ba|
```

Comment:
maybe make this `BankAccount.where(account_number: nil).or.where(bank_name: nil)` so it runs a little quicker if you have to run it more than once

Reply:
good call